src/pages/ProductList/Modal.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Modal.scss';
+
+const Modal = ({ open, onClose }) => {
+  if (!open) return null;
+
+  return (
+    <div className="overlay">
```

Comment:
```suggestion
    <div className="modal">
```
overlay보다는 컴포넌트의 이름으로 최상위 부모태그를 지정해주세요!